inject-generator/src/main/java/io/avaje/inject/generator/MethodReader.java

```diff
@@ -145,7 +202,7 @@ private String fullBuildMethod() {
   }
 
   String builderGetFactory() {
-    return String.format("      var factory = builder.get(%s.class);", factoryShortName);
+    return String.format("      var factory = builder.getNullable(%s.class); %n      if(factory == null) return;", factoryShortName);
```

Comment:
I'm thinking this does not have to be nullable as we don't get here if any of the checks fail. I'll check this.